### ISSUE TITLE
give an estimate of other team's stage

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2147]
When a player (or bot) sees an enemy, use that info to estimate other team's momentum. This could be useful for both players and bots.

**Comment by necessarily-equal** [comment]
Maybe? But if this is automatic then in practice we might as well just take the enemy team's momentum and tell the stage only, with maybe a bit of noise it would be much easier to implement IMO.

I think @cu-kai sent a screenshot about some trem mod showing the stages. I wanted to do something similar with some "team progress" that would represent how much you are winning – or loosing.

**Comment by bmorel** [comment]
I disagree with you because of the number of info you're giving. Too much.
My proposal is really about only saying "enemy reached stage N" after one of the team noticing it, which allows a team to play bluff.

**Comment by cu-kai** [comment]
The screenshot I sent was about BP usage, not stages.

**Comment by bmorel** [comment]
related to #2073
